### ISSUE TITLE
skimage submodules not imported correctly, regression tests fail

### ISSUE THREAD

**Issue by auroracramer** [marl/openl3#73]
skimage uses lazy imports, so we need to import each submodule explictly (e.g. `import skimage.transform; skimage.transform.rescale(X, s)` instead of `import skimage; skimage.transform.rescale(X, s)`).

**Comment by auroracramer** [comment]
It seems that after fixing it the regression tests start to fail: https://travis-ci.com/github/marl/openl3/jobs/529797736 :scream:

**Comment by auroracramer** [comment]
Fixing the call to [`skimage.transform.rescale`](https://scikit-image.org/docs/dev/api/skimage.transform.html#skimage.transform.rescale) to account for the channel dimension (which seems to be a newer change) gets rid of the first error, but the regression tests still fail.

**Comment by auroracramer** [comment]
I tried re-pinning `scikit-image<0.15.0` again, and the tests pass, so something is definitely up related to the upgrade.

**Comment by auroracramer** [comment]
From Travis:
```
    def test_image_regression(capsys):

        # test correct execution on test image file (regression)

        tempdir = tempfile.mkdtemp()

        run('image', DAISY_PATH, output_dir=tempdir, verbose=True)

    

        # check output file created

        image_outfile = os.path.join(tempdir, 'daisy.npz')

        assert os.path.isfile(image_outfile)

    

        # regression test

        image_data_reg = np.load(REG_DAISY_PATH)

        image_data_out = np.load(image_outfile)

        assert sorted(image_data_out.files) == sorted(image_data_reg.files) == ['embedding']

>       assert np.allclose(image_data_out['embedding'], image_data_reg['embedding'],

                           rtol=1e-05, atol=1e-05, equal_nan=False)

E       assert False

E        +  where False = <function debugallclose at 0x7ff003280400>(array([[-0.76532334,  3.5707924 , -0.01784389, ...,  2.2216794 ,\n         2.5874567 ,  0.31345925]], dtype=float32), array([[-0.831712  ,  3.2959087 , -0.15737686, ...,  2.1805274 ,\n         2.6374753 , -0.03956061]], dtype=float32), rtol=1e-05, atol=1e-05, equal_nan=False)

E        +    where <function debugallclose at 0x7ff003280400> = np.allclose

tests/test_regression.py:97: AssertionError
```

**Comment by auroracramer** [comment]
Thanks @beasteers for pointing out that the default values for `skimage.transform.rescale` changed!

**Comment by auroracramer** [comment]
Choosing `mode='constant'` and `anti_aliasing=False` seems to fix things, which is consistent with `skimage<0.15.0`

**Comment by auroracramer** [comment]
Closed by #75
